api/controllers/MerchStoreController.ts

```diff
@@ -188,4 +193,44 @@ export class MerchStoreController {
     await this.merchStoreService.updateOrderItems(fulfillOrderRequest.items);
     return { error: null };
   }
+
+  @Get('/pickupEvent/past')
+  async getPastPickupEvents(@AuthenticatedUser() user: UserModel): Promise<GetOrderPickupEventsResponse> {
+    const canSeePickupEventOrders = PermissionsService.canSeePickupEventOrders(user);
+    const pickupEvents = await this.merchStoreService.getPastPickupEvents(canSeePickupEventOrders);
```

Comment:
Sure, is the reasoning because converting the model to response type is the responsibility of the controller, since it's the one that produces a response? And that it shouldn't be the responsibility of the service layer?

Reply:
yeah